src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -287,6 +287,11 @@ protected int doWork() {
         IOUtil.assertFileIsWritable(OUTPUT);
         IOUtil.assertFileIsWritable(REJECT);
 
+        
+        if (CREATE_INDEX && DISABLE_SORT) {
```

Comment:
``
this type of input validation should be in customCommandLineValidation()
``

Reply:
in fact I realized that CREATED_INDEX is only for BAM-related tools, for LiftoverVCF, the index is always created event if CREATED_INDEX=false